crypto/err/openssl.txt

```diff
@@ -1,4 +1,4 @@
-# Copyright 1999-2023 The OpenSSL Project Authors. All Rights Reserved.
+# Copyright 1999-2021 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
mea culpa: this accidentally reverted the copyright year

Reply:
This does not matter. The update before release will handle this.